betting/src/test/betting/helpers/assert.mligo

```diff
@@ -0,0 +1,71 @@
+#import "../../../contracts/cameligo/betting/types.mligo" "Types"
+#import "../../../contracts/cameligo/betting/callback/main.mligo" "Betting_Callback"
+
+(* Assert contract result is successful *)
+let tx_success (res: test_exec_result) : unit =
+    match res with
+        | Fail (Rejected (error,_)) -> let () = Test.log(error) in failwith "Transaction should not fail"
+        | Fail _ -> failwith "Transaction should not fail"
+        | Success(_) -> ()
+
+(* Assert contract call results in failwith with given string *)
+let string_failure (res : test_exec_result) (expected : string) : unit =
+    let _expected = Test.eval expected in
+    match res with
+        | Fail (Rejected (actual,_)) -> assert (actual = _expected)
+        | Fail (Balance_too_low _) -> failwith "Contract failed: Balance too low"
+        | Fail (Other s) -> failwith s
+        | Success _ -> failwith "Transaction should fail"
+
+(* Assert Manager parameter with expected result *)
+let assert_balance (p_address : address) (expected : tez) : unit =
```

Comment:
`assert_` prefix is [unneedeed context](https://principles.dev/p/dont-add-unneeded-context/)

Reply:
Fixed